src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -288,7 +290,7 @@
      (fn []
        ~@body)))
 
-(defn with-query-results-cursor*
+(defn with-query-results-cursor
```

Comment:
Maybe we could do that in a separate PR, I kind of like it the way it is honestly, streamed-query-results isn't really similar to the java.jdbc functions